src/modules/coreinit/coreinit_exit.cpp

```diff
@@ -0,0 +1,24 @@
+#include "coreinit.h"
+#include "coreinit_exit.h"
+#include "utils/log.h"
+
+void
+_Exit()
```

Comment:
Functions with underscores are reserved for use in the standard library global namespace (both ones with lowercase and uppercase characters following the underscore)